WordPress/gutenberg
# Respect custom aspect ratio

## What?


Fixes #51929. Alternative to #52220

## Why?




## How?


1. Adds the width and height to the image in the figure as an inline style
2. Adds a deprecated save function
3. Updates the image width and height attributes to be strings.

## Testing Instructions





1. In a block editor
2. Insert an image block
3. Choose an image from the media library
4. In the block's inspector from the Aspect Ration panel input custom wifth and height
5. Make sure that scale is set to Cover
6. Save
7. Visit the front end
8. Both in the editor and the front end the image should be clipped by the custom aspect ratio

## Screenshots or screencast 

n/A
